cobalt/android/apk/app/src/app/assets/amati_device_inspector.js

```diff
@@ -0,0 +1 @@
+Android_AmatiDeviceInspector.printIsAmatiDevice();
```

Comment:
Last nitpick - technically this _may_ require a copyright header as well.  For JavaScript files it's okay to use the short form with SPDX identifier, like so:

```
/**
 * @license
 * Copyright The Cobalt Authors.
 * SPDX-License-Identifier: Apache-2.0
 */
```
Feel free to do this in a follow-up PR

Reply:
Updated